README.md

```diff
@@ -0,0 +1,12 @@
+# Warewulf
+
+Warewulf is a scalable systems management suite originally developed to manage
+large high-performance Linux clusters. Focused on general scalable system
+management, it includes a framework for system configuration, management,
+provisioning/installation, monitoring, event notification, and more via a modular
+plugin architecture. Install the components and features you need or leverage
+the existing system configurations stored within Warewulf to create custom
+solutions to meet your particular needs.
+
+This repository is a fork of https://github.com/warewulf/warewulf3.
```

Comment:
Can you take off the last paragraph, which appears to be specific to your work? Otherwise this is a good place holder for a README until we get the docs work merged in.